### DISCUSSION TITLE
Is there any documentation on the API?

### DISCUSSION THREAD

**Discussion by stevepresley** [Lolliedieb/lolMiner-releases#162] [General]
Basically, just what data we can expect to get back from the return, and are there any GET/PUT/POST endpoints (such as /Shutdown or ways to change parameters that were passed in at command line or via config file?  Is there a way to save config file with current running settings? etc)

I am working on a "watcher" desktop App, but if I could work on a node.js web app that could run on Linux/macOS/Windows, that would be a preferred solution.

Would be glad to help with documentation if none exists currently.

Thanks,
Steve

**Comment by Clankcoll** [comment]
Uuuuuuh that would be great.

Cause right now i have the problem that my miner crashes down to 262Kh/s and i would like to send atleast the Information what the actual hashrate is to my own server and monitor it with some audio alert if it drops below so that i can manually restart it

**Comment by jh442** [comment]
If you are using the command line, you can add --apiport <portnumber> to have the rig display nice JSON stats.  To access them, you have to browse to localhost:<portnumber>/summary.  Similarly, I do have this displayed to an interface I built to more easily access the state of all my GPUs.
